foundations/foundation-common/src/main/java/org/apache/servicecomb/foundation/common/net/NetUtils.java

```diff
@@ -145,12 +145,25 @@ public static IpPort parseIpPortFromURI(String uriAddress) {
     try {
       URI uri = new URI(uriAddress);
       String authority = uri.getAuthority();
-      return parseIpPort(authority);
+      return parseIpPort(uri.getScheme(), authority);
     } catch (URISyntaxException e) {
       return null;
     }
   }
 
+  private static IpPort parseIpPort(String scheme, String authority) {
+    if (authority == null)
+      return null;
+    int idx = authority.indexOf(':');
+    if (idx != -1)
+      return parseIpPort(authority);
+    if (scheme.equals("http"))
+      return parseIpPort(new StringBuffer(authority).append(":80").toString());
+    if (scheme.equals("https"))
+      return parseIpPort(new StringBuffer(authority).append(":443").toString());
+    return parseIpPort(authority);
```

Comment:
return null;

Reply:
only http and https was deal with here, if it's starts with others, e.g "rest://",shouldn't return null

Reply:
done